src/buildParams.ts

```diff
@@ -19,21 +22,25 @@ export const baseUrls: Record<string, string> = {
 // Update values here for the artist being featured
 export const projectDetails = {
   artistName: "MURS",
+  launchDate: "October 8, 2022 00:00:00",
```

Comment:
Would it be easier to get the data from the api in case it changes and we dont have to change it every release?

```
{
  "status": "error",
  "message": "Sale is closed!",
  "data": [
    "com.firehose.model.ProjectResponse",
    {
      "cost": 2000000,
      "tokenCostType": null,
      "tokenCost": null,
      "startDate": 1663373418000,
      "endDate": null,
      "dripCampaigns": null
    }
  ]
}

```